Gemfile

```diff
@@ -6,4 +6,4 @@ gem 'jekyll-sitemap'
 
 gem 'jekyll-assets'
 gem 'jekyll-redirect-from'
-
+gem 'bootstrap', '~> 4.4.1'
```

Comment:
@sgha229 What was this added for? Is this PR the migration to BS4 as well?

Reply:
@raney24 Yes, this is part of the BS4 migration. I had planned to just release this alongside the other changes as well. Is this an issue?